app/assets/javascripts/Components/Result/url_viewer.jsx

```diff
@@ -5,15 +5,16 @@ export class URLViewer extends React.Component {
     super(props);
     this.state = {
       url: "",
```

Comment:
Do we need to store the url in state anymore? Can't we just use the `this.props.externalUrl` value instead?

Reply:
We can. This was a design decision though since `this.props.externalUrl` can be a string that says the file cannot be loaded. I thought therefore it was cleaner to store urls specifically in the state since other ways I thought of of flagging an unreadable URL wasn't as readable.